backend/sample.env

```diff
@@ -120,8 +120,8 @@ SESSION_COOKIE_AGE=86400
 # Time in seconds to wait before polling LLMWhisperer's status API
 ADAPTER_LLMW_POLL_INTERVAL=30
 # Total number of times to poll the status API.
-# 500 mins to allow 1500 (max pages limit) * 20 (approx time in sec to process a page)
-ADAPTER_LLMW_MAX_POLLS=1000
+# ~60 mins (assuming it'll be enough to process 1500 pages with LLMW v2)
+ADAPTER_LLMW_MAX_POLLS=120
```

Comment:
@chandrasekharan-zipstack but what about for the V1? Doesn't it use the same ENV?

Reply:
@ritwik-g you're right, v1 uses the same envs. Shall we discourage support for v1 by changing this env (reducing the time / max possible pages) and instead enforce support with v2 for more number of pages?
Realistically speaking - I doubt if any user has such large extraction times. 

Worst case, 
1. either we'll have to let this env be and take action after we sunset v1
1. or introduce a new set of envs for v2 and update that (involves changes in the SDK, so I'm not a fan of this)